docs/network-spec/mux.tex

```diff
@@ -132,12 +132,12 @@ \subsection{Flow-control and Buffering in the Demultiplexer}
 %\item Shutdown the threads that run the mini protocols.
 %\item Close Socket/ free resources.
 %\end{itemize}
-%
-%\newcommand{\Larval}{\state{Larval}}
-%\newcommand{\Connected}{\state{Connected}}
-%\newcommand{\Mature}{\state{Mature}}
-%\newcommand{\Dying}{\state{Dying}}
-%\newcommand{\Dead}{\state{Dead}}
```

Comment:
Was this bug introduced in previous commit? The doc in `master` builds so I guess this can be squashed where the bug was introduced in this pr.